tests/src/system/basic/WskBasicTests.scala

```diff
@@ -87,48 +87,69 @@ class WskBasicTests
         stdout should include regex ("""(?i)whisk API version\s+v1""")
     }
 
+    it should "validate default property values" in {
+        val wskpropsBackup = TestUtils.backupWskProps()
```

Comment:
I don't anticipate that we'll remove it or if we do, we will replace it with a command line argument that points to a wskprop file --- I added this support for the following reasons: I have several config files for multiple scenarios/deployments and testing. So I rapidly switch profiles this way.

I think it's a mistake for tests to muck around with user files - we should avoid it. The concern about committing wskprops into source control could be mitigated in other ways. For example, we can mandate that the prop file is always called `.wskprops` and the value of WSK_CONFIG_FILE becomes the path to that file. Then a gitignore file could do the rest.